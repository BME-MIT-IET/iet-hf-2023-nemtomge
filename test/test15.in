loadmap map.json 
start V1 
set Scientist V1 PL1 
give Gear V1 Gloves 
give Gear V1 Axe 
give Gear V1 Bag 
set Gear PL1 Coat 
show Gear V1
touch V1 
1 
2 
show Gear V1
exit
