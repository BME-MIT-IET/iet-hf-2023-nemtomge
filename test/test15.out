Map from file map.json loaded
The game has been started. Players are: V1 
Scientist V1 set on field PL1 
Scientist V1 has been given gear Gloves 
Scientist V1 has been given gear Axe 
Scientist V1 has been given gear Bag 
Gear Coat has been placed on field PL1 
V1's gears are:
Gloves(3) 
Axe(1) 
Bag 
Choose one to pick: 
0 - Cancel
1 - Coat
Choose one to drop: 
0 - Cancel
1 - Gloves(3)
2 - Axe(1)
3 - Bag
V1's gears are:
Gloves(3) 
Bag 
Coat
